spoonacular-kotlin/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.beetlestance.spoonacular_kotlin">
```

Comment:
do we not have namespace support for ava/kotlin module?

Reply:
With new AGP version, a warning was coming to use namespaces instead package name

Reply:
The manifest should not have been present in the Kotlin module in the first place, resoled the issue.